src/metabase/public_settings.clj

```diff
@@ -273,6 +274,12 @@
   :type       :json
   :default    {})
 
+(defsetting application-font
```

Comment:
should this verify the set value against `available-fonts`? See `site-locale`'s setter for prior art.